packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -58,3 +58,10 @@ export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
     </FileCardGroup>
   );
 };
+
+const DownloadIconTrampoline = (): JSX.Element => {
```

Comment:
I had trouble writing utility code to avoid these trampoline functions at point-of-use.

This works for now... and follows the principle of smallest conditionally compiled code.